### PR TITLE
DE updated 1.5

### PR DESCRIPTION
**Pull request by zRxnx**
I hope its now correctly xD

### CODE REVIEW

File: language/de.json [hunk 1]
```diff
@@ -173,8 +173,8 @@
 
     "forceeasteregg": "Easter Egg aktivieren",
 
-    "permissioneditor": "Permission Editor",
-    "permissioneditorguide": "Permissions auf dem Server editieren, speichert nach ~g~easyadmin_permissions.cfg~w~",
+    "permissioneditor": "Berechtigung's Editor",
```
Comment (Blumlaut) [inline language/de.json:176]: `Berechtigung*'*s"?

Comment (zRxnx) [reply language/de.json:176]: Yes, thats right.

Comment (Blumlaut) [reply language/de.json:176]: `'s` denotes that its "someones", Berechtigung is not who's Editor it is, its what kind of Editor it is.

**Comment by zRxnx** [comment]
But i can rename the german "Berechtigung" to "Permission" back, because you cant rename "principal, ACE, ...." to german, yk?

**Comment by Blumlaut** [comment]
ACE and Principal is not really a thing that directly translates into German, you could maybe see if the NL Translation translated it.

**Comment by zRxnx** [comment]
Ok i look

**Comment by zRxnx** [comment]
Ok they also dont rename "Permissions" to their language but they renamed the editor, i look what i can do

**Comment by zRxnx** [comment]
Have a look on 1.5.1, should be better

**Pull request merged**